client/constants/interfaces.ts

```diff
@@ -102,43 +102,58 @@ export type KeyValuePair = {
   [key: string]: any;
 };
 
-export interface BroaderReviewValues {
+interface BaseReviewValues {
   overallComments: string;
   finalScore: number;
-  projectTypeScore: string;
-  projectNeedScore: number;
-  projectFundingScore: number;
-  pastBcaapFundingScore: number;
-  facilityMasterPlanScore: number;
-  facilityUsageScore: number;
-  trafficDataScore: number;
-  climatePerspectiveScore: number;
-  climateBestPracticesScore: number;
-  environmentalRisksScore: number;
-  environmentalInnovationScore: number;
-  projectDescriptionScore: number;
-  climateGoalsScore: number;
-  organizationClimateGoalScore: number;
-  successMeasurementScore: number;
-  safetyScore: number;
-  medevacScore: number;
-  localBenefitsScore: number;
-  longTermScore: number;
-  communitySupportScore: number;
-  contingencyPlanScore: number;
-  classBCostScore: number;
-  thirdPartyContributionScore: number;
   completionStatus: string;
 }
 
+export interface NetworkReviewValues extends BaseReviewValues {
+  reasonableCostforCommunitySize: number;
+  s3ComponentsScore: number;
+  s4DescribeHowATNPAlignsWithCommunityGoals: number;
+  s4DescribePotentialEconomicBenefits: number;
+  s5DetailsHowATNPWillAddressSafetyConcerns: number;
+  s6DescribeConsultationUndertaking: number;
+  s6DescribeDataCollectionUndertaking: number;
+  s6DescribeHowATNPImplementationWillEnsureSuccess: number;
+  fundingReceivedLastFiveYears: number;
+}
+
+export interface ReviewValues extends BaseReviewValues {
+  projectTypeScore: string;
+  projectNeedScore: string;
+  projectFundingScore: string;
+  pastBcaapFundingScore: string;
+  facilityMasterPlanScore: string;
+  facilityUsageScore: string;
+  trafficDataScore: string;
+  climatePerspectiveScore: string;
+  climateBestPracticesScore: string;
+  environmentalRisksScore: string;
+  environmentalInnovationScore: string;
+  projectDescriptionScore: string;
+  climateGoalsScore: string;
+  organizationClimateGoalScore: string;
+  successMeasurementScore: string;
+  safetyScore: string;
+  medevacScore: string;
+  localBenefitsScore: string;
+  longTermScore: string;
+  communitySupportScore: string;
+  contingencyPlanScore: string;
+  classBCostScore: string;
+  thirdPartyContributionScore: string;
+}
+
 export interface ScoreSummaryTableProps {
   applicationId: number;
 }
 
 export interface BroaderReviewScore {
   createdAt: Date;
   updatedAt: Date;
-  data: BroaderReviewValues;
+  data: ReviewValues;
```

Comment:
Does it need to be `NetworkReviewValues | InfrastructureReviewValues`? I don't think we match `ReviewValues`

Reply:
In fact - I don't see `BroaderReviewScore` interface being referenced anywhere. Do we need it at all?

Reply:
Nope, youre right, theres a few spots I may have missed from the merge, thanks for catching them!

Reply:
> In fact - I don't see `BroaderReviewScore` interface being referenced anywhere. Do we need it at all?

It's being reference in the Score summary table